_includes/index/beta-banner.html

```diff
@@ -0,0 +1,9 @@
+<div class="row justify-content-center">
+    <div class="col-12 text-center">
+        <div class="display-4 text-black fw-bold">
+            <p class="lead text-red mb-0">
+                This website is under active development and considered beta. Release: December 2024 
```

Comment:
The above looks like this:
<img width="800" alt="image" src="https://github.com/elixir-europe-training/ELIXIR-Training-SPLASH/assets/23480589/5fccdb21-c84a-4ea1-88d6-4294cefce8bc">

I would propose the following changes
```suggestion
<div class="bg-warning" style="position: relative; left: 50%; width: 100vw; transform: translate(-50%);">
    <div class="text-center">
        <div class="display-4 fw-bold p-4">
            <p class="lead text-red mb-0">
                This website is under active development and considered beta. <br> Planned release: December 2024
```

to obtain this look:
<img width="800" alt="image" src="https://github.com/elixir-europe-training/ELIXIR-Training-SPLASH/assets/23480589/46eca59a-274e-4849-b220-4e01ff537930">

Reply:
yes, that looks very nice. Thanks @mihai-sysbio